Dockerfile

```diff
@@ -11,8 +11,10 @@ ARG TARGETPLATFORM=${TARGETPLATFORM}
 ARG BUILDPLATFORM=${BUILDPLATFORM}
 
 # Ruby image to use for base image, change with [--build-arg RUBY_VERSION="3.3.x"]
+# renovate: datasource=docker depName=node depName=docker.io/ruby
 ARG RUBY_VERSION="3.3.2"
 # # Node version to use in base image, change with [--build-arg NODE_MAJOR_VERSION="20"]
+# renovate: datasource=node-version depName=node
 ARG NODE_MAJOR_VERSION="20"
```

Comment:
Is the preset correctly handling the fact that this variable only contains the major version?

Or should we switch this to `NODE_VERSION` and have a specific version, as it would now be bumped by Renovate automatically? iirc we used a major version here to avoid having to bump the version for each node release.

@vmstan any opinion?

Reply:
I would think that Renovatebot would only track the major version if it is the only major version. But I think we'll have to wait for Renovatebot to actually crawl to be sure.

Reply:
In my tests it will trigger an update to Node 22 as the major version, which I have no strong preference if we leave it that way or switch to be more specific. Leaving it as it would reduce churn of those PRs.